sa3ed_app/www/found_person/index.js

```diff
@@ -0,0 +1,25 @@
+document.getElementById('foundPersonForm').addEventListener('submit', function(event) {
```

Comment:
we are using VueJS with frappe, it would be easier for you to interact with the backend apis using `frappe.call()` instead of the AJAX calls.